easybuild/easyconfigs/d/deal.II/deal.II-9.3.3-foss-2021a.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'CMakeMake'
+
+name = 'deal.II'
+version = '9.3.3'
+
+homepage = 'https://www.dealii.org'
+description = """deal.II is a C++ program library targeted at the computational solution of
+ partial differential equations using adaptive finite elements."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+source_urls = ['https://github.com/dealii/dealii/releases/download/v%(version)s/']
+sources = ['dealii-%(version)s.tar.gz']
+checksums = ['5dfb59174b341589e92b434398a1b7cc11ad053ce2315cf673f5efc5ba271a29']
+
+builddependencies = [('CMake', '3.20.1')]
+
+dependencies = [
+    ('Boost', '1.76.0'),
+    ('GSL', '2.7'),
+    ('HDF5', '1.10.7'),
+    ('METIS', '5.1.0'),
+    ('netCDF', '4.8.0'),
+    ('p4est', '2.8'),
+    ('PETSc', '3.15.1'),
+    ('zlib', '1.2.11'),
+]
+
+configopts = "-DCMAKE_BUILD_TYPE=Release "
+configopts += "-DDEAL_II_WITH_LAPACK=ON -DDEAL_II_WITH_MPI=ON "
+configopts += "-DDEAL_II_WITH_BOOST=ON -DBOOST_DIR=$EBROOTBOOST"
+configopts += "-DDEAL_II_WITH_GSL=ON -DGSL_DIR=$EBROOTBOOST"
+configopts += "-DDEAL_II_WITH_HDF5=ON -DHDF5_DIR=$EBROOTBOOST"
+configopts += "-DDEAL_II_WITH_METIS=ON -DMETIS_DIR=$EBROOTBOOST"
+configopts += "-DDEAL_II_WITH_NETCDF=ON -DNETCDF_DIR=$EBROOTBOOST"
+configopts += "-DDEAL_II_WITH_P4EST=ON -DP4EST_DIR=$EBROOTP4EST"
+configopts += "-DDEAL_II_WITH_PETSC=ON -DPETSC_DIR=$EBROOTBOOST"
```

Comment:
This looks wrong. Almost all of those are pointing at `$EBROOTBOOST`. And they are missing a space from the end of the line, so they'll get concatenated into one long string.

Reply:
Ouch, yeah, I clearly wasn't paying attention much there... Will fix.

Reply:
It looks like you based in on the previous one and that needs correcting as well - see #15692